src/Shepard/Storage/LocalStorage.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Shepard\Storage;
+
+use Symfony\Component\Process\Process;
+
+class LocalStorage implements StorageInterface
+{
+    /**
+     * @var string
+     */
+    private $basePath;
+
+    /**
+     * @param string $basePath
+     */
+    public function __construct($basePath = "/tmp/")
+    {
+        $this->basePath = $basePath;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function storeContent($path, $content)
+    {
+        if (!is_dir($this->basePath)) {
+            (new Process(mkdir($this->basePath, 0755, true)))->run();
```

Comment:
mkdir-t nem kell process-el futtatni, egyébként ez valami bool, vagy nem is tudom mi lehet a visszatérése.
ne tehát ez itt nem fura, hanem hülyeség